/**:
  ros__parameters:
    dir_logs: "Desktop/logs"
    dirs_sub:
      - "flir"
    names_actors:
      - "flir2"
    topics:
      - "/flir2/image_raw"